plugins/techdocs/src/reader/components/TechDocsReaderPageContent/dom.tsx

```diff
@@ -233,22 +250,22 @@ export const useTechDocsReaderDom = (
               --md-code-bg-color: ${theme.palette.background.paper};
               --md-code-hl-color: ${alpha(theme.palette.warning.main, 0.5)};
               --md-code-hl-keyword-color: ${
-                isDarkTheme
+                theme.palette.type === 'dark'
```

Comment:
Curiosity: any specific reason to this change?

Reply:
The theme change was done to avoid dependencies on `useEffect` not being remembered between re-renders like `isDarkTheme` was before... Sorry for not explaining in the pull request description, I've added a code comment now.